src/lib/Helper/ParameterHelper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Helper;
+
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+class ParameterHelper
```

Comment:
Could you please explain what is the reason for creating this helper? I can see only four usages of `parseParameters` method, all of them in the controllers. As for me, it looks like unnecessary copying the same data all around. You can use the original `ParameterBag`, in this case, `$request->query` in all of these places. Am I missing something?